### DISCUSSION TITLE
PayNow Payment Method

### DISCUSSION THREAD

**Discussion by kirinnee** [juspay/hyperswitch#3378] [Ideas / Feature Requests]
Would like to see PayNow (SG's top Local Payment Method), which has linkage to Malaysia's DuitNow and India's UPI, on hyperswitch.

https://abs.org.sg/consumer-banking/pay-now

**Comment by VedantKhairnar** [comment]
Thanks for the suggestion.

We are really fascinated to see the linkage of PayNow with UPI and DuitNow. But our bandwidth have been dedicated for North Americas and European Union as the target geographies for Q1 2024.

While this is being said, we would be happy to accept or support any code contribution to add PayNow in Hyperswitch and expand the coverage.
